src/bulma/pages/products/components/BaseForm.vue

```diff
@@ -1,12 +1,22 @@
 <template>
     <enso-form class="box form-box has-background-light raises-on-hover"
         @ready="form = $event.form">
-        <template v-slot:category_id="{ field }">
+        <template v-slot:category_id="{ field, errors }">
             <label class="label">
                 {{ i18n('Category') }}
+                <span class="icon is-small has-text-info"
+                    v-tooltip="i18n(field.meta.tooltip)"
+                    v-if="field.meta.tooltip">
+                    <fa icon="info-circle"
+                        size="xs"/>
```

Comment:
I somebody need to add a tooltip (small chances) here would have to customize the Form object. In that case he can also customize the view. Let's remove it for now